2-mandatory/6-budgets.js

```diff
@@ -11,6 +11,9 @@ Should give return the answer of 62600.
 **/
 
 function getBudgets(peopleArray) {
+  let amountOfBudgets = 0;
+  peopleArray.forEach(person => amountOfBudgets += person.budget);
+return amountOfBudgets;
 }
```

Comment:
Hi Simon! Like the way your solution is so neat and short. I did this exercise using a longer route.